pkg/services/ngalert/state/template/error.go

```diff
@@ -0,0 +1,16 @@
+package template
+
+import (
```

Comment:
nit: I'd inline this error. I don't see a need for a new file - more indirection more files to look at.

Reply:
Inlined! 🙂